tox.ini

```diff
@@ -159,3 +159,12 @@ commands =
 whitelist_externals =
     rm
     {[testenv]whitelist_externals}
+
+[testenv:changelog]
+skip_install = false
+usedevelop = false
+description = Generate the changelog
+extras =
```

Comment:
I'd rather not install all Molecule's dependencies when just looking at fragments by a third-party tool is needed...